pom.xml

```diff
@@ -63,6 +63,7 @@
         <jboss.logging.tools.wf8.version>1.2.0.Final</jboss.logging.tools.wf8.version>
         <jboss-jaxrs-api_2.0_spec>1.0.0.Final</jboss-jaxrs-api_2.0_spec>
         <jboss.spec.javax.xml.bind.jboss-jaxb-api_2.2_spec.version>1.0.4.Final</jboss.spec.javax.xml.bind.jboss-jaxb-api_2.2_spec.version>
+        <org.json.version>20080701</org.json.version>
```

Comment:
Please don't use org.json we use fasterxml/jackson and don't want to json libraries